### ISSUE TITLE
Helm version for 1.0.2

### ISSUE THREAD

**Issue by blaketastic2** [istio/istio#8546]
**Describe the bug**
Helm chart version wasn't updated with the 1.0.2 release, even though there were changes mentioned in the release notes

**Comment by obeyler** [comment]
```
helm package istio
Successfully packaged chart and saved it to: /data/shared/istio-1.0.2/install/kubernetes/helm/istio-1.0.1.tgz

```

**Comment by hklai** [comment]
@sdake I suppose we need to automate something like https://github.com/istio/istio/pull/8091?

**Comment by sdake** [comment]
@hklai its a 1 liner shell operation - just not sure how to get the operation into the release workflow.

Cheers
-steve

**Comment by hklai** [comment]
@sdake could you elaborate? What is that shell operation?

**Comment by sdake** [comment]
@hklai I left  a comment in the commit log:

```
commit 9f29f6fbe08788f191413053e8d96c37eac99a56
Author: Steven Dake <stdake@cisco.com>
Date:   Mon Aug 27 11:09:58 2018 -0700

    Revise version to 1.1.0 (#8241)

    command for further time travelers:

    find ./ -type f -exec sed -i -e 's/1\.0\.0/1.1.0/g' {} \;
```

That sed command should probably be enhanced to search for Version: and appVersion: and find should operate on install/kubernetes/helm directory children.

**Comment by sdake** [comment]
@hklai my recommendation would be to execute that find operation right after a tag is created - then commit the change for the first commit of every new z-stream.on the release-1.0 branch.

I just don't know where to automate this process.

Cheers
-steve

**Comment by hklai** [comment]
Unfortunately we could not do anything for 1.0,2 now, but this had been fixed in 1.0.3.

Closing this
